redcap/project.py

```diff
@@ -425,6 +425,9 @@ def export_records(
             "exportCheckboxLabel",
         )
 
+        if export_survey_fields:
+            fields = fields.extend([name + "_complete" for name in self.forms])
+
```

Comment:
Doesn't `list.extend` modify the list in place? So it would be
```suggestion
            fields.extend([name + "_complete" for name in self.forms])

```

Reply:
Yes, it does.

And now I'm terribly confused, since the return value of list.extend is None, and so this should break instantly - but I've been using it in the field for months, installing directly from the GitHub repo every time.

I shall do some investigation when I get into the office this morning!

Reply:
Hey @forsakendaemon -- any update on this?

Reply:
Hi @pwildenhain - I apparently hadn't done what I needed to to fix this (apologies, I'm still learning Pull Requests). It should be up to date and able to be pulled in now, I believe.